packages/react-components/src/components/table/types.ts

```diff
@@ -14,6 +14,8 @@ export type TableItem = {
   [key in string]: TableItemRef | Primitive | undefined;
 };
 
+export type Quality = 'GOOD' | 'BAD' | 'UNCERTAIN';
```

Comment:
There's already a type defined for this. You can just access it off the data point - `DataPoint['quality']`

Reply:
Done, thanks